### ISSUE TITLE
Solution for Arabic language for this amazing plugin

### ISSUE THREAD

**Issue by learn05** [dompdf/dompdf#2841]
Hello All, 
and thanks for the dompdf Developer,
I want to provide the arabic solution for dompdf for laravel, 
the Steps as below:
<ol>
  <li>Create a Folder inside App Called Library.</li>
  
  <li>Upload the <a href="https://github.com/learn05/arabiclang/">Repo </a>to the Library Folder.</li>
  
  <li>Open the text.php file, that is located in:<br>
    <code>\vendor\dompdf\dompdf\src\Renderer</code>
  </li>
    
  <li>find the below almost in line number 90 line.<br>
    <code>$this->_canvas->text($x, $y, $text,
            $font, $size,
            $style->color, $word_spacing, $char_spacing);
    </code>
  </li>
  
  <li>Add the Below Code Above it:<br>
    <code>
    if (strtolower($style->direction) == 'rtl') {
        $Arabic = new \App\Libraries\I18N\I18N_Arabic('Glyphs');
        $text = $Arabic->utf8Glyphs($text);
        }
    </code>
  </li>
  
</ol>
the repo solution is this link <a href="https://github.com/learn05/arabiclang/"> Click Here</a>

**Comment by TahaAboudaya** [comment]
Yes, the problem of the letters has been fixed and shown correctly, but there is a problem with the space. Did you find the solution?

**Comment by ayaseensd** [comment]
`$style->direction) == 'rtl'`  will never return true if you didn't add direction property in your html to be rendered.
So, add 
```
<style type="text/css" media="screen">
    
            html {
                direction: rtl;
                }
</style>
```

**Comment by Magnatox** [comment]
> Hello All, and thanks for the dompdf Developer, I want to provide the arabic solution for dompdf for laravel, the Steps as below:
> 
>     1. Create a Folder inside App Called Library.
> 
>     2. Upload the [Repo ](https://github.com/learn05/arabiclang/)to the Library Folder.
> 
>     3. Open the text.php file, that is located in:
>            `\vendor\dompdf\dompdf\src\Renderer`
> 
>     4. find the below almost in line number 90 line.
>            `$this->_canvas->text($x, $y, $text,
>                    $font, $size,
>                    $style->color, $word_spacing, $char_spacing);
>            `
> 
>     5. Add the Below Code Above it:
>            `
>            if (strtolower($style->direction) == 'rtl') {
>                $Arabic = new \App\Libraries\I18N\I18N_Arabic('Glyphs');
>                $text = $Arabic->utf8Glyphs($text);
>                }
>            `
> 
> 
> the repo solution is this link [ Click Here](https://github.com/learn05/arabiclang/)

Hello @learn05 !
How do you get rid of the white spaces before the actual text starts?
